app/models/user.rb

```diff
@@ -565,6 +565,47 @@ def reactions_to
     Reaction.for_user(self)
   end
 
+  def latest_activity
```

Comment:
I wasn't sure what we'd consider as "latest activity" for a user, but figured that the [below](https://github.com/forem/forem/pull/17033/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR571) covered most, if not all, of what we'd want. Any comments or concerns about my approach to this?

cc @Ridhwana @aitchiss @anujbhavsar96

Reply:
There is a [slack conversation](https://forem-team.slack.com/archives/C7GE84DEJ/p1645481144902869) that Jennie started for the same when we were looking to add this metric to the table. I hope this helps.

Reply:
Thanks, @anujbhavsar96!